app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,39 @@
+<html>
+  <head>
+    <%= stylesheet_link_tag 'user_new' %>
+
+  </head>
+
+  <body>
+
+    <div class=classContainer>
+
+      <h2>Log in to Blog APP</h2>
+
+      <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+        <div class="field">
+          <%= f.label :email %>
+          <%= f.email_field :email, autofocus: true, autocomplete: "email", class:'input' %>
+        </div>
+
+        <div class="field">
+          <%= f.label :password %>
+          <%= f.password_field :password, autocomplete: "current-password", class:'input' %>
+        </div>
+
+        <% if devise_mapping.rememberable? %>
+          <div class="field">
+            <%= f.check_box :remember_me %>
+            <%= f.label :remember_me %>
+          </div>
+        <% end %>
+
+        <div class="actions">
+          <%= f.submit "Log in", class:'btn'  %>
+        </div>
+      <% end %>
+
+      <%= render "devise/shared/links" %>
+    </div>
+  </body>
+</html>
```

Comment:
- [ ] You did a great job with styling 👏👏 However, as suggested, please ensure that your design matches the wireframe. In the wireframe, labels are not used for input fields; instead, placeholders are used ⛏ Could you please fix this? 😀

Current design | Wireframe
---| ---|
![image](https://github.com/smunoz1988/blog-on-rails/assets/106119105/c6eeeead-9b34-43a9-8383-3e06f3d075cd) | ![image](https://github.com/smunoz1988/blog-on-rails/assets/106119105/44450fb4-ae9f-414f-b669-de3a46dfdb83)

Reply:
@KanzaTahreem  Hello!

I understand your request but the requirements says that I can add my own style but following the wireframe:

![image](https://github.com/smunoz1988/blog-on-rails/assets/64698693/0d8d4139-1e08-4833-aa54-b4a73e9dee73)

![image](https://github.com/smunoz1988/blog-on-rails/assets/64698693/cc17c5c7-cbc6-4d01-a861-568b9958bec1)

Can you explain me please, because I follow the wireframe and add my own style.